windows/ReactNativeWebView/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.UI.Xaml" version="2.8.0-prerelease.210927001" targetFramework="native" />
+  <package id="Microsoft.Web.WebView2" version="1.0.1018-prerelease" targetFramework="native" />
```

Comment:
I believe adding an exact WebView2 version here has broken app scenarios where a different version of WebView2 is being installed via the WinUI package. See https://github.com/react-native-webview/react-native-webview/pull/2459

Reply:
Sure, I can drop this :)